pkg/contextstore/store_test.go

```diff
@@ -0,0 +1,119 @@
+package contextstore
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"gotest.tools/assert"
+)
+
+func TestStoreInitScratch(t *testing.T) {
+	dirname, err := ioutil.TempDir("", "test-store-init-scratch")
+	assert.NilError(t, err)
+	defer os.RemoveAll(dirname)
+	testDir := filepath.Join(dirname, "test")
+	_, err = NewStore(testDir)
+	assert.NilError(t, err)
+	metaFS, err := os.Stat(filepath.Join(testDir, metadataDir))
+	assert.NilError(t, err)
+	assert.Equal(t, metaFS.IsDir(), true)
+	tlsFS, err := os.Stat(filepath.Join(testDir, tlsDir))
+	assert.NilError(t, err)
+	assert.Equal(t, tlsFS.IsDir(), true)
+
+	// check that we can create a store from existing dir
+	_, err = NewStore(testDir)
+	assert.NilError(t, err)
+}
+
+func TestSetGetCurrentContext(t *testing.T) {
+	testDir, err := ioutil.TempDir("", "TestSetGetCurrentContext")
+	assert.NilError(t, err)
+	defer os.RemoveAll(testDir)
+	store1, err := NewStore(testDir)
+	assert.NilError(t, err)
+	err = store1.SetCurrentContext("test")
+	assert.NilError(t, err)
+	store2, err := NewStore(testDir)
+	assert.NilError(t, err)
+	assert.Equal(t, "test", store2.GetCurrentContext())
+}
+
+func TestExportImport(t *testing.T) {
+	testDir, err := ioutil.TempDir("", "TestSetGetCurrentContext")
```

Comment:
This is the wrong name, and also differs from other places where you use-hyphen-separated-style which ought to be consistent (at least within the file).

Personally I think you should just use `t.Name()` everywhere, although I observe that is not the norm in this repo.